src/Native/System.Net.Http.Native/pal_easy.h

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include "pal_types.h"
+
+#include <curl/curl.h>
+
+#define CurlOptionLongBase 0
+#define CurlOptionObjectPointBase 10000
+#define CurlOptionFunctionPointBase 20000
+
+enum PAL_CURLoption : int32_t
+{
+    PAL_CURLOPT_INFILESIZE = CurlOptionLongBase + 14,
```

Comment:
A question here, since we are in the native world here, can't we just say ```PAL_CURLOPT_INFILESIZE = CURLOPT_INFILESIZE``` ? Is there any advantage of expanding the macro here?